test/python_rules/importlib_metadata_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+
+class ImportlibMetadataTest(unittest.TestCase):
+    def test_can_detect_version(self):
+        # This fails  if .dist-info isn't  available for wheels downloaded from
+        # pip
+        from importlib_metadata import version
+
+        self.assertEqual(version("importlib_metadata"), "0.23")
+
+    def test_nonexistant_module(self):
+        from importlib_metadata import distribution, PackageNotFoundError
+
+        with self.assertRaises(PackageNotFoundError):
+            distribution("nonexistant")
```

Comment:
Might be nice to have a test for distribution that doesn't raise `PackageNotFoundError`?

Reply:
The first test do this by calling `importlib_metadata.version()` which internally call `distribution().version`.

Reply:
Ah, great, thanks